### ISSUE TITLE
Bug with running pipelines

### ISSUE THREAD

**Issue by jacobrosenthaldfci** [Dana-Farber-AIOS/pathml#181]
When running a preprocessing pipeline, sometimes some of the tile images are lost.
This only happens with some tiles, and I'm not sure why it happens to those tiles and not others. 

example:
````python
wsi = HESlide("../data/CMU-1.svs", name = "example")
pipeline = Pipeline() # pass-thru pipeline, does not change tiles
out_TESTING = wsi.run(pipeline, distributed=False, level=1)

### View results

fix, ax = plt.subplots(ncols = len(out_TESTING), nrows = 2, figsize=(10, 4))
for i, (tile1, tile2) in enumerate(zip(out_TESTING, wsi.tiles)):
    ax[0, i].imshow(tile1.image)
    ax[1, i].imshow(tile2.image.astype(np.uint8))
for a in ax.ravel(): 
    a.axis("off")
plt.tight_layout()
````

![Screen Shot 2021-08-30 at 11 48 52 PM](https://user-images.githubusercontent.com/53529525/131438711-160fde70-c869-4178-b088-306db0d13400.png)

I think this isn't a problem with the pipeline or the tile, because it works when I just apply the pipeline to the tiles that failed:

````python
fig, ax = plt.subplots(nrows = 1, ncols = 2)

ax[0].imshow(out_TESTING[1].image)
ax[1].imshow(pipeline.apply(out_TESTING[1]).image)

ax[0].axis("off"); ax[1].axis("off");
````
![Screen Shot 2021-08-30 at 11 51 06 PM](https://user-images.githubusercontent.com/53529525/131438862-6f367b81-1677-4d8d-aff2-cf03a6d37c2d.png)

And I don't think it's a dask problem, since I used `distributed=False`. So my guess is maybe it's an h5 problem?

Need to figure out why it is failing for those tiles in particular.

**Comment by jacobrosenthaldfci** [comment]
I think I've identified the problem (or at least *a* problem). OpenSlide uses (x, y) coordinates, but we are using (i, j) coordinates for everything else, so we need to make sure to swap the order whenever interacting with OpenSlide. I thought I had accounted for this but I guess not completely. 
Will push a fix to `dev-issue181` branch

**Comment by jacobrosenthaldfci** [comment]
Closed in #184
